scripts/core/editor3/components/Editor3.jsx

```diff
@@ -28,7 +28,6 @@ Editor3Base.propTypes = Editor3Component.propTypes = {
 const mapStateToProps = (state) => ({
     readOnly: state.readOnly,
     showToolbar: state.showToolbar,
-    highlightsEnabled: state.allowsHighlights,
```

Comment:
There was the option to disable highlights from config file. Why did you dropped this feature.

Reply:
The feature was only working forward, meaning once you enable it, disabling wouldn't work properly  - you could no longer add new highlights, but existing ones would be visible, but nothing would happen if you clicked it.

I thought since now highlights can be configured for any field, we might not need to hardcode it in the file itself. If we still need this feature I can restore it, but we'd need to fix it.